src/main/resources/api-definition.yaml

```diff
@@ -49,6 +49,74 @@ schemas:
 
   - requisitionLineItemDto: !include schemas/requisitionLineItemDto.json
 
+  - approveRequisitionDto: |
+      {   "type": "object",
+          "$schema": "http://json-schema.org/draft-04/schema",
+          "title": "ApproveRequisitionDto",
+          "description": "A single requisitionDto for batch approval",
+          "properties": {
+              "id": { "type": "string", "title": "id" },
+              "modifiedDate": {"type": "array", "title": "modifiedDate", "items": {"type": "integer"}, "uniqueItems": false },
```

Comment:
@palbecki I remember it used to return an array of numbers in the past in the JSON representation. Is that not the case anymore?